examples/expo-kit/test/mocks/react-native-safe-area-context.js

```
@@ -0,0 +1,5 @@
+const React = require('react');
```

ESLint

[error] A `require()` style import is forbidden.

(`@typescript-eslint/no-require-imports`)

---

[error] 'require' is not defined.

(no-undef)